app/views/shared/_proportion-protected-chart.html.erb

```diff
@@ -16,7 +16,7 @@
           <p class="card__habitat-title"><%= dataset[:name] %></p>
           <p class="card__habitat-status">
             <% if @habitats_present %>
-              <% if (dataset_status(dataset) == 'unknown' || dataset_status(dataset) == 'absent') %>
+              <% if GeoEntity::NEGATIVE_OCCURRENCE_STATUSES.include?(dataset_status(dataset)) %>
```

Comment:
Could this be turned into a helper that takes dataset as the argument? Might be able to use it elsewhere too. Don't worry if that requires a lot of refactoring!

GeoEntity::NEGATIVE_OCCURRENCE_STATUSES.include?(dataset_status(dataset))

Reply:
Yep I'm going to place this in the Countries Helper for now. To be honest, I think we need a presenter at this rate to deal with the amount of logic in the view files.